odoo/addons/base/models/ir_cron.py

```diff
@@ -570,6 +576,7 @@ def _update_failure_count(self, job, status):
             job['id'],
         ])
 
+    @api.model
```

Comment:
`@api.model` on a private method. xmo is gonna argue that it is useless because the decorator only serves for xmlrpc, and private methods cannot be called via xmlrpc.

Reply:
Yes and it also means that the method does not use records in self.